frontend/src/router/index.ts

```diff
@@ -30,9 +30,14 @@ const router = createRouter({
     },
     {
       path: '/profile',
-      name: 'profile',
+      name: 'myProfile',
       component: ProfileView
-    }
+    },
+		{
+			path: '/profile/:nick',
```

Comment:
Not sure if it’s using nick property in user but it should user intra_nick, nick can be changed